crates/external/rpc/src/traits.rs

```diff
@@ -6,20 +6,33 @@ use alloy_primitives::Log;
 use alloy_sol_types::SolEvent;
 use async_trait::async_trait;
 use core::time::Duration;
-use jsonrpsee::core::ClientError;
+use jsonrpsee::{core::ClientError, types::ErrorObjectOwned};
 use kona_interop::{CROSS_L2_INBOX_ADDRESS, ExecutingMessage, SafetyLevel};
 use tokio::time::error::Elapsed;
 
+const SAFETY_START_MSG: &str = "(safety level: ";
+const UNKNOWN_CHAIN_MSG: &str = "unknown chain: ";
+const MINIMUM_SAFETY_MSG: &str = "does not meet the minimum safety";
 /// Failures occurring during validation of [`ExecutingMessage`]s.
 #[derive(thiserror::Error, Debug)]
 pub enum ExecutingMessageValidatorError {
+    /// Message does not minimum safety level
+    #[error("Message does not meet safety level. Expected: {0}, Got: {1}")]
+    MinimumSafety(SafetyLevel, SafetyLevel),
+    /// Invalid chain
+    #[error("Unsupported chainId: {0}")]
+    UnknownChain(u64),
```

Comment:
```suggestion
    /// Invalid chain
    #[error("unsupported chain id: {0}")]
    UnknownChain(u64),
```
all error message should be lowercase
ref: https://doc.rust-lang.org/std/error/trait.Error.html